lib/matplotlib/axis.py

```diff
@@ -640,8 +640,36 @@ class Ticker(object):
     formatter : `matplotlib.ticker.Formatter` subclass
         Determines the format of the tick labels.
     """
-    locator = None
-    formatter = None
```

Comment:
I hope that removing these (public) class attributes isn't going to trip anyone up.  Maybe that would involve such an obscure use case that it won't happen.

Reply:
We did just have that issue, but in this case I suspect it will be ok.

These lines go back to d78e6c498a0cde2d5063eadc63addd8eb36b0953 and I suspect it was done done to make sure that `Ticker` sub-classes always had the attributes.  Putting something into these attributes at the class level does not really make sense.